blog/info/templates/info/faqs.html

```diff
@@ -0,0 +1,128 @@
+
+{% extends "base.html" %}
+<!--extend the functionality from the header of the parent component.-->
+{% block head %}
+    {{ super() }}
+    <link rel= "stylesheet" type= "text/css" href= "{{ url_for('static',filename='css/faqs.css') }}">
+    <link href="https://fonts.googleapis.com/css2?family=Amatic+SC:wght@700&family=Francois+One&family=Kalam:wght@700&display=swap" rel="stylesheet">
+{% endblock %}
+
+{% block content %}
+<div class="container" id="accordion">
+    <div class="row">
+        <div class="col-10">
+            <h1>Frequently asked questions</h1>
+            <p>Here are answers to some questions you guys might have for me..</p>
+        </div>
+    </div>
+    <div class="card">
```

Comment:
a good idea here would be to break the card into it's own partial and then pass the unique bits to that partial to simplify this :)